routemaster/services/codec.rb

```diff
@@ -19,6 +19,10 @@ def load(data)
         _log.warn { "Failed to decode message" }
         _log_exception(e)
         Models::Message::Garbled.new
+      rescue TypeError => e
+        _log.warn { "Failed decoding message. Code #{code}, Data #{data}, Hash #{hash}" }
```

Comment:
Nit: If we can use code=code, data=data format, until we move to logging with JSON those fields will be parsed by Datadog.

Reply:
Sure, does this look better?

`"Failed decoding message. code=#{code}, data=#{data}, hash=#{hash}"`

Reply:
![Image](https://media.giphy.com/media/3ohzdFViL2QatSyUF2/source.gif)

Reply:
Strange choice of GIF, TBH.

Reply:
You know, Giphy.